modules/dnn/src/tflite/tflite_importer.cpp

```diff
@@ -163,6 +163,8 @@ void TFLiteImporter::populateNet()
     CV_Assert(modelTensors);
     layouts.resize(modelTensors->size(), DATA_LAYOUT_UNKNOWN);
     size_t subgraph_inputs_size = subgraph_inputs->size();
+    std::vector<std::string> inputsNames;
+    std::vector<MatShape> inputsShapes;
```

Comment:
vector sizes are known here. It makes sense to define size and do not use push_back.

Reply:
Sure, thanks